packages/mdc-fab/_mixins.scss

```diff
@@ -89,12 +89,14 @@
 }
 
 @mixin mdc-fab-shape-radius($radius, $rtl-reflexive: false) {
-  @include mdc-shape-radius(mdc-shape-resolve-percentage-radius($mdc-fab-height, $radius), $rtl-reflexive);
+  @include mdc-shape-radius($radius, $rtl-reflexive);
 
   &.mdc-fab--mini {
-    @include mdc-shape-radius(mdc-shape-resolve-percentage-radius($mdc-fab-mini-height, $radius), $rtl-reflexive);
+    @include mdc-shape-radius($radius, $rtl-reflexive);
   }
+}
 
+@mixin mdc-fab-extended-shape-radius($radius, $rtl-reflexive: false) {
```

Comment:
Is the `&.mdc-fab--extended` selector now redundant since this is only expected to be used within that context?

Reply:
Makes sense.

Also, added not() selector for regular/mini fab variant so extended variant don't get extra styles.